docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -1,4 +1,4 @@
-.. _slurm-known-issues:
+.. _known-hpc-issues:
```

Comment:
I didn't see a reference to this -- could be elsewhere I suppose.

Reply:
There isn't a reference to that tag, the page is currently referenced by page name (slurm-known-issues) in overview.rst.

I added it in case we wanted to reference the page as a whole, but I wanted to be able to use the tag name slurm-known-issues (as opposed to the file name) to reference the specific section.

Reply:
It would really make sense to rename all these pages to hpc-xxx, but I didn't want to take that on at the moment.